docs/asyncio_blocking_operations.md

```diff
@@ -95,6 +95,22 @@ urllib does blocking I/O and should be run in the executor with the standard met
 
 `os.stat` does blocking disk I/O and should be run in the executor with the standard methods above.
 
+#### write_bytes
+
+`write_bytes` does blocking disk I/O and should be run in the executor with the standard methods above.
```

Comment:
I'd prefix all these method names with `pathlib.Path` so it's clear where the method comes from.

Reply:
https://github.com/home-assistant/developers.home-assistant/compare/pathlib?expand=1